CODEOWNERS

```diff
@@ -11,6 +11,7 @@ app_dart/lib/src/request_handlers/dart_internal_subscription.dart           @dre
 app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart              @keyonghan
 app_dart/lib/src/request_handlers/flush_cache.dart                          @keyonghan
 app_dart/lib/src/request_handlers/get_build_status.dart                     @keyonghan
+app_dart/lib/src/request_handlers/get_build_status.dart                     @CaseyHillers
```

Comment:
Seems a duplicated entry was added.

Reply:
Whoops, thanks for the catch! Sent https://github.com/flutter/cocoon/pull/3366